d2l-floating-buttons.html

```diff
@@ -111,9 +115,9 @@
 				window.addEventListener('scroll', this._reposition);
 
 				this._isRTL = (getComputedStyle(this._container).direction === 'rtl');
-				this._reposition();
 
 				Polymer.RenderStatus.afterNextRender(this, function() {
+					this._reposition();
```

Comment:
Yes. Otherwise the  buttons don't float when first rendered.

I think in Polymer 2 attached is called sooner than in Polymer 1